tmt/base.py

```diff
@@ -475,14 +475,30 @@ def lint(self):
 
         # Check if the format of Markdown file respects the specification
         # https://tmt.readthedocs.io/en/latest/spec/tests.html#manual
-        md_path = tmt.export.return_markdown_file()
-        if os.path.exists(md_path):
-            invalid_md_file = tmt.export.check_md_file_respects_spec(md_path)
-            if invalid_md_file:
-                for i in invalid_md_file:
-                    verdict(False, i)
-            else:
-                verdict(True, "correct headings are used in the Markdown file")
+        if self.manual:
+            try:
+                tests_amount = len(self.test.split())
+            except AttributeError:
```

Comment:
Not necessary. Each test has to have the `test` attribute defined otherwise it would not be detected as test.